### PR TITLE
fix(frontend, backend): add link lifetime to mail + proper message

### PR DESCRIPTION
**Pull request by grignards**
See #4759

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

**Testing method**

- Start the backend with:
```shell
cd backend
docker-compose up -d
export TRACIM_EMAIL__NOTIFICATION__FROM__EMAIL=test@test.te
export TRACIM_EMAIL__NOTIFICATION__REPLY_TO__EMAIL=test@test.te
export TRACIM_EMAIL__NOTIFICATION__REFERENCES__EMAIL=test@test.te
export TRACIM_EMAIL__NOTIFICATION__ACTIVATED=True
export TRACIM_EMAIL__NOTIFICATION__SMTP__SERVER=localhost
export TRACIM_EMAIL__NOTIFICATION__SMTP__PORT=1025
export TRACIM_EMAIL__NOTIFICATION__SMTP__USER=user
export TRACIM_EMAIL__NOTIFICATION__SMTP__PASSWORD=password
pserve development.ini
```
- Request a password reset through the UI
- Go to http://localhost:8025 to read the password reset mail.

### CODE REVIEW

File: frontend/i18next.scanner/pt/translation.json [hunk 1]
```diff
@@ -473,5 +473,6 @@
   "Alex Doe": "Carmo Da Silva",
   "alex.doe@jco.co": "carmo.dasilva@cds.pt",
   "In order to finish your sign in, please read the following usage conditions and accept them": "Para terminar a sua sessão, por favor leia as seguintes condições de utilização e aceite-as",
-  "Unknown content": "Conteúdo desconhecido"
+  "Unknown content": "Conteúdo desconhecido",
+  "Your reset password link has expired, you can regenerate one by following <1>this link</1>.": "O ligação da sua senha de reinicialização expirou, pode regenerar um, clicando <1>nesta ligação</1>."
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/pt/translation.json:477]:
```suggestion
  "Your reset password link has expired, you can regenerate one by following <1>this link</1>.": "A ligação da sua senha de reinicialização expirou, pode regenerar uma nova clicando <1>nesta ligação</1>."
```

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: frontend/i18next.scanner/pt/translation.json (hunk 1 above)
Comment (grignards) [reply frontend/i18next.scanner/pt/translation.json]: Applied directly with git as github doesn't let me apply the suggestion through its interface…

**Review by gbonaspetti — APPROVED** [review]
Ok for frontend part

**Review by inkhey — APPROVED** [review]
Backend is ok.

**Pull request merged**